src/main/java/org/b612foundation/adam/opm/OdmFormatter.java

```diff
@@ -9,6 +9,7 @@
  */
 public final class OdmFormatter {
   private static final String CCSDS_OPM_VERS = "CCSDS_OPM_VERS";
+  private static final String CCSDS_OEM_VERS = "CCSDS_OEM_VERS";
```

Comment:
Duplicate?

Reply:
Nope. One is "OPM" and the other is "OEM". Both in the string and the variable name.

Reply:
Oops! It appears I can't read :)